packages/containers/src/ActionBar/ActionBar.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+
+import { ActionBar as Component } from 'react-talend-components';
+import Container from './ActionBar.container';
+import Connected, {
+	mapDispatchToProps,
+} from './ActionBar.connect';
+
+describe('Component ActionBar', () => {
+	it('should render', () => {
+		const wrapper = renderer.create(
+			<Component />
+		).toJSON();
+		expect(wrapper).toMatchSnapshot();
+	});
+});
+
+describe('Container ActionBar', () => {
+	it('should render', () => {
+		const wrapper = shallow(
+			<Container />
+		);
+		expect(wrapper).toMatchSnapshot();
+	});
+});
+
+describe('Connected ActionBar', () => {
```

Comment:
If we remove the connector (see review on connector), don't forget to remove this